core/persist/Persistable.ts

```diff
@@ -8,11 +8,14 @@ import {isEqual} from 'lodash';
 
 /**
  * Interface for objects that can be bound to a {@link PersistenceProvider}.
- * @typeParam S - must be serializable to JSON and PersistableState<S> must be observable
+ * @typeParam S - must be serializable to JSON and PersistableState<S> must be observable.
+ * @param defaultPersistableState - (optional) sets the default persistable state. If not set, the
+ * PersistenceProvider will save an initial snapshot of `getPersistableState()` as the default state.
  */
 export interface Persistable<S> {
     getPersistableState(): PersistableState<S>;
     setPersistableState(state: PersistableState<S>): void;
+    defaultPersistableState?: PersistableState<S>;
```

Comment:
This gives the user full control of the `defaultPersistableState` without the need to create an exotic `getPersistableState()` method.